shoop/simple_supplier/module.py

```diff
@@ -58,3 +58,7 @@ def update_stock(self, product_id):
         if latest_event:
             sv.stock_value_value = latest_event.purchase_price_value * sv.logical_count
         sv.save(update_fields=("logical_count", "physical_count", "stock_value_value"))
+
+    def get_last_purchase_price(self, product_id):
```

Comment:
This is returning ``StockAdjustment``-object not a purchase price. I think this method is also missing from default module (which should return zero-price).